src/frontend/styles.css

```
@@ -1,6 +1,7 @@
 @import "tailwindcss";
 @import "highlight.js/styles/github-dark.css";
 @plugin "@tailwindcss/typography";
+@source "../node_modules/streamdown/dist/*.js";
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)